test/testapps/BasicTestApp/RenderFragmentToggler.cshtml

```diff
@@ -7,7 +7,7 @@
         @ExampleFragment
     }
 
-    <button onclick=@{ showFragment = !showFragment; }>Toggle</button>
+    <button onclick=@(x => showFragment = !showFragment)>Toggle</button>
```

Comment:
Super minor nitpicky preference: for anyone reading this code, the `x` might look a bit mysterious. I had the impression that `_` was the convention for "I don't care about this arg".